.github/labeler.yml

```diff
@@ -18,4 +18,5 @@ connector-api-change:
   - openapi/connector*
 documentation:
   - docs/**/*
```

Comment:
will the `docs/**/* also be like "docs/**/*"

Reply:
For docs it is not needed because only the *.md that was causing issues